examples/js/loaders/GLTFLoader.js

```diff
@@ -763,7 +763,19 @@ THREE.GLTFLoader = ( function () {
 
 		var extension = textureDef.extensions[ name ];
 		var source = json.images[ extension.source ];
-		var loader = source.uri ? parser.options.manager.getHandler( source.uri ) : parser.textureLoader;
+
+		var loader;
+		if ( source.uri ) {
+
+			loader = parser.options.manager.getHandler( source.uri );
+
+		}
+
+		if ( ! loader ) {
```

Comment:
I think `} else {` would be better here.

Reply:
Then the logic would be same as before. The intent here is to handle the case `getHandler` returns a `null` value which is not tested in the above condition.

Reply:
Ah I see... May be good to add a quick comment then?

Reply:
It's the same pattern as used in [this part of the code in the same file](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/loaders/GLTFLoader.js#L2435-L2449). Should I add a comment in both places?

Reply:
The main issue is the `! loader`. I'm not a fan of variables that can be `undefined` or `null`.

How about this?

```js
		var loader = parser.textureLoader;

		if ( source.uri ) {

			var handler = parser.options.manager.getHandler( source.uri );
			if ( handler !== null ) loader = handler;

		}
```

Reply:
I'm fine with it -- I was just trying to follow the already established pattern in the file. Do you want me to change both places?

Reply:
Yes please 🙏